src/components/Matching/styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components'
+import Button from '../Button'
+
+export const Wrapper = styled.section`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+`
+
+export const TitleTabs = styled.div`
+	display: flex;
+	width: 100%;
+
+	& a {
+		flex-grow: 1;
+	}
+`
+
+export const ContentTabs = styled.div`
+	box-sizing: border-box;
+	width: 100%;
+	padding: 1em;
+	border-bottom-left-radius: 0.8em;
+	border-bottom-right-radius: 0.8em;
+	background-color: ${({ theme }) => theme.text.link};
```

Comment:
Interesting choice of colour, it really catches the eye! Suits our most important functionality well. :sunflower:

Reply:
☺️